WEB/src/components/prononciation/Verbalize.vue

```diff
@@ -0,0 +1,344 @@
+  const grandAcces = () => {
+    navigator.mediaDevices
+      .getUserMedia({ audio: true, video: false })
+      .then(handleSuccess)
+      .catch((error: any) => {
+        console.log(error)
+      })
+  }
+
+  // Quand on clic sur le bouton du micro soit on démare l'enregistrement soit on le coupe
+  const recordClicked = () => {
+    if (mediaRecorder.value?.state == 'recording') {
+      // On coupe le record
+      mediaRecorder.value?.stop()
+      recording.value = false
+      recordingState.value.style.visibility = 'hidden'
+      debug(mediaRecorder.value.state)
+      debug(dataList.value)
+      playerDisplay.value = true
+
+      // On met un set timeout pour qu'il ai bien le temps de mettre les valeurs dans dataList
+      // On va prendre l'ArrayBuffer contenu dans le blob et le reconvertir en .wav (car de base on peut uniquement record en .webm avec js)
+      setTimeout(() => {
+        debug(audioContext.value)
+        debug(mediaRecorder.value)
+        debug(dataList.value[0])
+        dataList.value[0].arrayBuffer().then((response: ArrayBuffer) => {
+          debug(response)
+          audioContext.value?.decodeAudioData(response).then((audioBuff: AudioBuffer) => {
+            const arrayBuffer: ArrayBuffer = encodeWavFileFromAudioBuffer(audioBuff, 0)
+            debug(arrayBuffer)
+
+            // On crée un nouveau Blob à partir de ces données
+            soundBlob.value = new Blob([arrayBuffer], { type: 'audio/wav' })
+            debug(soundBlob.value)
+            const audioUrl: string = window.URL.createObjectURL(soundBlob.value)
+
+            // On assigne le blob en temps que source de la balise audio pour que l'utilisateur puisse s'écouter
+            soundName.value = audioUrl
+            player.value!.src = soundName.value
+
+            // On crée un nouveau fichier à partir des informations contenues dans le Blob
+            // soundData.value = new File([audioUrl.value], 'record')
+
+            // On réinitialise le tableau de données
+            dataList.value = []
+          })
+        })
+      }, 1)
+      // Si le record n'est pas lancé on rentre ici
+    } else {
+      // On lance l'enregistrement
+      // Quand les données sont disponibles => Donc quand on arrête l'enregistrement, on met les données dans la liste de données "dataList"
+      mediaRecorder.value?.start()
+
+      recording.value = true
+      goodResponse.value = false
+      badResponse.value = false
+      errorResponse.value = false
+      recordingState.value.style.visibility = 'visible'
+
+      debug(mediaRecorder.value?.state)
+
+      if (mediaRecorder.value) {
+        mediaRecorder.value.ondataavailable = (e: any) => {
+          dataList.value.push(e.data)
+        }
+      }
+    }
+  }
+
+  const sendToDjango = () => {
+    let formData = new FormData()
+    formData.append('data', soundBlob.value!, soundName.value)
+    formData.append('word', 'bestelling')
+
+    axios
```

Comment:
Personal recommendation : *Try async/await request you will not be disappointed*